fvtest/porttest/omrstrTest.cpp

```diff
@@ -1130,13 +1131,10 @@ TEST(PortStrTest, str_convU8ToPlat_Null)
 /**
  * Verify string wide->MUTF8 conversion, source string has no byte order mark.
  */
-TEST(PortStrTest, str_convWideToU8Bom)
+TEST(PortStrTest, DISABLED_str_convWideToU8Bom)
```

Comment:
Why do these tests need to be disabled?

I believe they used to pass before this functionality was refactored into OMR. Has a regression been introduced at some point?